tests/test_examples.py

```diff
@@ -131,19 +78,17 @@ def test_from_and_to_json_roundtrip(source, filename) -> None:
         raise AssertionError(msg) from err
 
 
-@pytest.mark.filterwarnings(
-    "ignore:DataFrameGroupBy.apply.*:DeprecationWarning",
-)
-@pytest.mark.parametrize(("source", "filename"), distributed_examples, ids=id_func)
-@pytest.mark.skipif(
-    not VL_CONVERT_AVAILABLE,
-    reason="vl_convert not importable; cannot run mimebundle tests",
-)
-def test_render_examples_to_png(source, filename) -> None:
+@slow
+@ignore_DataFrameGroupBy
+@distributed_examples
+@skip_requires_vl_convert
+def test_render_examples_to_png(source: Any, filename: str) -> None:
```

Comment:
Just highlighting somewhere these new markers improve readability.

The difference quite significant in an editor, as you can see the docstrings on hover - but no clutter at the test site